exploits/minitwlpayload/bootloader/source/boot.c

```diff
@@ -66,6 +67,10 @@ extern unsigned long _start;
 extern unsigned long argStart;
 extern unsigned long argSize;
 
+// Manually define as true. DSI exploits always run in DSi mode.
+// Will return as undefined during compile if not defined this way. (with current setup)
+bool dsiMode = true;
```

Comment:
So it should be this?:

static const dsiMode = 1;


I will make the suggested changes.